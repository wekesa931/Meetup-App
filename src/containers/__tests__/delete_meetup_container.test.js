import React from "react";
import { Provider } from "react-redux";
import Enzyme, { shallow } from "enzyme";
import configureMockStore from "redux-mock-store";
import DeleteContainer from '../create_container';
import EnzymeAdapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new EnzymeAdapter() });

const mockStore = configureMockStore();
const store = mockStore({});

describe('Create Meetup container', () => {
    const wrapper = shallow(<Provider store={store}>
                                <DeleteContainer />
                            </Provider>);
    it('renders correctly', () => {
        expect(wrapper).toMatchSnapshot()
    })
})
