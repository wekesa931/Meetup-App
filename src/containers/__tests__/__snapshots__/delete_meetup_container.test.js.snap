// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Create Meetup container renders correctly 1`] = `ShallowWrapper {}`;
